src/Tabs.tsx

```diff
@@ -1 +1,52 @@
-// implement a component here
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from './Tab';
+import './Tabs.scss';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: React.Dispatch<React.SetStateAction<Tab>>;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  let selectedTab = null;
+
+  return (
+    <div className="tabs">
+      {tabs.map(tab => {
+        const isTabSelected = selectedTabId === tab.id;
+
+        if (isTabSelected) {
+          selectedTab = tab;
+        }
+
+        return (
+          <button
+            key={tab.id}
+            type="button"
+            className={classNames(
+              'tabs__button',
+              { 'tabs__button--active': isTabSelected },
+            )}
+            onClick={() => {
+              if (!isTabSelected) {
+                onTabSelected(tab);
+              }
+            }}
```

Comment:
looks like a handler function needs to be created

Reply:
done